.github/workflows/tests.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         run: |
           conda activate pyjanitor-dev
           python -m pip install -e .
-          pytest -m "${{ matrix.test-subset }}"
```

Comment:
Later we could update these to the Makefile.

Reply:
And we also could split normal test cases and doctest cases.
Currently, they are mixed up.

Reply:
splitting doctests tests and other tests were done at #1205